source/how-tos/enable-intel-tsn.rst

```diff
@@ -0,0 +1,311 @@
+
+   <a href="https://slimbootloader.github.io/security/container-security.html?highlight=container#container-security" target="_blank">Container Security</a>
+
+
+SBL defines a container called IPFW for settings related to TSN and TSN is managed as a component inside this IPFW container.
+SBL loads this container and processes it as required including passing the relevant settings to Intel® Firmware Support Package (Intel® FSP) during Runtime.
+SBL defines this container in each platform's build config file BoardConfig.py.
+
+**TSN MAC Address**
+
+User needs to provide TSN MAC address that is unique per board and that can be independently updateable. FBU (Firmware and BIOS Utility) tool is used to create a raw TSN MAC binary.
+This raw TSN binary needs to be signed using the component key that is used during SBL build so that it could pass SBL verification during boot time.
+The signed TSN component is kept into SBL container called IPFW and encapsulate it to create a capsule image. Capsule image is then signed by a firmware update key.
+
+**NOTE:** The firmware update key should match with SBL key so that the capsule image could pass SBL capsule verification during firmware update.
+
+Capsule location can be specified by SBL configuration data. SBL will update the TSN component inside the IPFW container using the specified capsule image during  next boot once firmware update is triggered.
+
+Details on how this is all generated and packaged will be discussed in the later sections.
+
+
+**TSN enable**
+
+For each platform, there is a build configuration file BoardConfig.py. User can check this build configuration  file to make sure self.ENABLE_TSN_MAC_ADDRESS is set to 1 to enable TSN MAC Address binary.
+The other TSN related configuration data is PchTsnEnable. Make sure to set this to 1 under Silicon Configuration Data.
+
+Take Tiger Lake as an example.
+
+  https://github.com/slimbootloader/slimbootloader/blob/master/Platform/TigerlakeBoardPkg/BoardConfig.py::
+
+    self.ENABLE_TSN_MAC_ADDRESS = 1
+    if self.ENABLE_TSN_MAC_ADDRESS:
+        self.TMAC_SIZE = 0x00001000
+
+    # Name | Image File     | CompressAlg |         AuthType          |                 Key File                   | Region Align   | Region Size  | Svn Info
+    # ===============================================================================================================================================================
+    ('IPFW',  'SIIPFW.bin',      ''    ,     container_list_auth_type,   'KEY_ID_CONTAINER'+'_'+self._RSA_SIGN_TYPE,          0,          0      ,        0),   # Container Header
+    ('TMAC',     ''       ,     'Lz4'  ,     container_list_auth_type,   'KEY_ID_CONTAINER_COMP'+'_'+self._RSA_SIGN_TYPE,     0,   self.TMAC_SIZE,        0),   # TSN MAC Address
+
+In the build configuration file, user can set ``self.ENABLE_TSN_MAC_ADDRESS`` to 1 to enable TSN component and adjust the size if needed. The TSN container name is``IPFW`` by default.

```

Comment:
The TSN container name is``IPFW`` by default.- remove this line. covered already